### PR TITLE
Create jeananns1.json

### LINKED ISSUE #5987: I'm confused
Hello! :wave: Please add me as a collaborator to this repo.  
Why Is this different from the training.  Very confusing.

### PR DESCRIPTION
**Pull request by jeananns1**
Added my pin per @githubteacher  closes #5987

### CODE REVIEW

File: _pinx/jeananns1.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: jeananns1
```
Comment (brianamarie) [inline _pinx/jeananns1.json]: @jeananns1 Try adding these characters to the beginning of the file and the end of the file `---` like in the example. Also, make sure the title is `_pins/`, it looks like there may be a spelling error. Let me know if you want any help!

Comment (jeananns1) [inline _pinx/jeananns1.json]: I figured out the missing text, but can't figure out how to edit the file name.

Comment (brianamarie) [inline _pinx/jeananns1.json]: screen shot 2017-04-28 at 1 25 52 pm
When you edit the file, try clicking on the top by the title and erasing. [Here are some step by step instructions(https://help.github.com/articles/renaming-a-file/).

**Comment by jeananns1** [comment]
@githubteacher  What Is wrong now?

File: _pinx/jeananns1.json (hunk 1 above)
Comment (jeananns1) [inline _pinx/jeananns1.json]: Now what is wrong?  Also it is not obvious that I had to erase the name before I could get to the sub-folder.

**Comment by brianamarie** [comment]
@jeananns1 It looks like there was a 'trailing space' on one of the lines, which I removed. I'll keep an eye on the tests. Thank you for being so patient throughout this process!

**Comment by davidc4991vvc** [comment]
@jeananns1  Good job looks good!

**Comment by brianamarie** [comment]
@davidc4991vvc Please do not close other people's pull requests or issues 😄

**Comment by jeananns1** [comment]
Adding my map pin to the master branch

**Comment by brianamarie** [comment]
Hi @jeananns1! This looks good, go ahead and merge whenever you're ready!

**Comment by eggnaro** [comment]
I did it :P

**Pull request merged**